package.json

```diff
@@ -34,6 +38,7 @@
     "url": "https://github.com/jenkinsci/jenkins-design-language.git"
   },
   "dependencies": {
+    "harp": "^0.20.3",
```

Comment:
will this be used by other stuff in blue ocean? (will it be brought along as a transitive dep?)